airbyte-webapp/src/components/connection/CatalogTree/next/CatalogTreeTableRow.module.scss

```diff
@@ -31,20 +36,35 @@
     cursor: pointer;
   }
 
-  &.disabledChange {
-    background-color: colors.$red-50;
+  &.removed {
```

Comment:
A mixin could make this repeated pattern easier to generate for all the statuses